### ISSUE TITLE
Create auth token for user after account creation

### ISSUE THREAD

**Issue by Alex3917** [GetBlimp/django-rest-framework-jwt#124]
Let's say I want to create an auth token and return it to the user after they successfully create an account. Is the best way to do this to just redirect the user to the obtain_jwt_token endpoint, or is there some other way that would be preferred in this case?

**Comment by Alex3917** [comment]
edit: I see someone answered this on Stack Overflow:

http://stackoverflow.com/questions/27529690/how-to-do-django-json-web-token-authentication-without-forcing-the-user-to-re-ty

Maybe this should be added to the docs?

**Comment by jpadilla** [comment]
@Alex3917 yeah that looks good. I made a tweak on 116c8b2d08c0e471d9fe6643682fded072e64975 which should make it even simpler.

Want to put a PR together to add to our docs? How about under an FAQ section?

**Comment by Alex3917** [comment]
@jpadilla 

Sounds good, I'll create a pull request in a couple days.

**Comment by Alex3917** [comment]
I created a pull request here:

https://github.com/GetBlimp/django-rest-framework-jwt/pull/163
